### PR TITLE
feat: add Dutch translations

### PR DESCRIPTION
**Pull request by SoulaAndrikopoulos**
# Description of change
This PR adds Dutch translations.

## Links to any relevant issues
N/A.

## How the change has been tested
I am a native Dutch speaker. 

## Definition of Done checklist
Add an `x` to the boxes that are relevant to your changes.

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/i18n/nl/index.ts [hunk 1]
```diff
@@ -185,10 +185,10 @@ const nl = {
       BADGES: 'Badges',
     },
     EMPTY_CREDENTIALS: {
-      TITLE: '',
-      SUBTITLE: '',
+      TITLE: 'Het is hier een beetje stil',
```
Comment (Norlock) [inline src/i18n/nl/index.ts]: Ik zou dit iets formeler weergeven: Geen credentials gevonden. Ik denk dat wat beter is voor onze doelgroep

File: src/i18n/nl/index.ts [hunk 2]
```diff
@@ -185,10 +185,10 @@ const nl = {
...
+      SUBTITLE: 'Waarom voeg je geen nieuwe credentials toe om te starten met je digitale "me"?',
```
Comment (Norlock) [inline src/i18n/nl/index.ts]: Hier misschien: Voeg credentials toe om je digitale "me" te starten.

File: src/i18n/nl/index.ts [hunk 3]
```diff
@@ -204,26 +204,26 @@ const nl = {
...
+      EMPTY: 'Nog geen data.',
     },
   },
   TIMELINE: {
-    EMPTY: '',
+    EMPTY: 'Nog geen activiteit.',
   },
   SEARCH: {
-    INPUT_PLACEHOLDER: '',
+    INPUT_PLACEHOLDER: 'Zoek voor iets',
```
Comment (Norlock) [inline src/i18n/nl/index.ts]: Zoeken of Zoeken naar

File: src/i18n/nl/index.ts [hunk 4]
```diff
@@ -204,26 +204,26 @@ const nl = {
...
     NO_QUERY: {
-      TITLE: '',
-      DESCRIPTION: '',
+      TITLE: 'Waar zullen we zoeken?',
```
Comment (Norlock) [inline src/i18n/nl/index.ts]: Weet niet wanneer dit getoond wordt. Maar denk: Geef een zoekopdracht op.

File: src/i18n/nl/index.ts [hunk 5]
```diff
@@ -87,89 +87,89 @@ const nl = {
...
-      DESCRIPTION: '',
-      CONFIRM: '',
-      CANCEL: '',
+      TITLE: 'Reset app',
+      DESCRIPTION: 'Weet je zeker dat je de app wilt resetten en alle gegevens wilt verwijderen? ',
+      CONFIRM: 'Ja, reset alles',
+      CANCEL: 'Nee, bewaar mijn profiel',
     },
-    ACCOUNT: '',
+    ACCOUNT: 'Account',
```
Comment (Norlock) [inline src/i18n/nl/index.ts]: Profiel

File: src/i18n/nl/index.ts [hunk 6]
```diff
@@ -87,89 +87,89 @@ const nl = {
...
+      TITLE: 'Mijn profiel',
       PROFILE_NAME: {
-        TITLE: '',
-        NAVBAR_TITLE: '',
-        INPUT_PLACEHOLDER: '',
-        CONFIRM: '',
+        TITLE: 'Profielnaam',
+        NAVBAR_TITLE: 'Bewerk profielfoto',
+        INPUT_PLACEHOLDER: 'Profielnaam invoeren',
+        CONFIRM: 'Updaten',
```
Comment (Norlock) [inline src/i18n/nl/index.ts]: Wijzigingen opslaan

File: src/i18n/nl/index.ts [hunk 7]
```diff
@@ -87,89 +87,89 @@ const nl = {
...
       },
       DISPLAY_PICTURE: {
-        TITLE: '',
-        EDIT: '',
-        CHANGE: '',
+        TITLE: 'Toon afbeelding',
+        EDIT: 'Bewerk',
+        CHANGE: 'Updaten',
```
Comment (Norlock) [inline src/i18n/nl/index.ts]: Opslaan

File: src/i18n/nl/index.ts [hunk 8]
```diff
@@ -87,89 +87,89 @@ const nl = {
...
-        TITLE: '',
-        BUTTON_TEXT: '',
+        TITLE: 'Onboarding journey',
+        BUTTON_TEXT: 'Herstart',
       },
       HINTS_AND_TIPS: {
-        TITLE: '',
-        BUTTON_TEXT: '',
+        TITLE: 'Hints en tips',
+        BUTTON_TEXT: 'Reset',
```
Comment (Norlock) [inline src/i18n/nl/index.ts]: Terugzetten / Opnieuw instellen

File: src/i18n/nl/index.ts [hunk 9]
```diff
@@ -87,89 +87,89 @@ const nl = {
...
+        NUMBER: 'Nummers',
+        CHARACTERS: 'Karakters',
       },
     },
     RESET_APP: {
-      TITLE: '',
-      DESCRIPTION: '',
-      CONFIRM: '',
-      CANCEL: '',
+      TITLE: 'Reset app',
```
Comment (Norlock) [inline src/i18n/nl/index.ts]: Terugzetten app of Opnieuw instellen app

File: src/i18n/nl/index.ts [hunk 10]
```diff
@@ -87,89 +87,89 @@ const nl = {
...
       },
       HINTS_AND_TIPS: {
-        TITLE: '',
-        BUTTON_TEXT: '',
+        TITLE: 'Hints en tips',
+        BUTTON_TEXT: 'Reset',
       },
       DEVELOPER_MODE: {
-        TITLE: '',
+        TITLE: 'Developer modus',
```
Comment (Norlock) [inline src/i18n/nl/index.ts]: Ontwikkelaarsmodus

**Review by Norlock — COMMENTED** [review]
Er zitten nog wel wat engelse woorden in, maar misschien vind Jelle dat niet zo erg

File: src/i18n/nl/index.ts (hunk 1 above)
Comment (SoulaAndrikopoulos) [reply src/i18n/nl/index.ts]: Goede suggestie! De engelse versie is misschien wat informeler:
 TITLE: "It's a bit quiet in here",
      SUBTITLE: 'Why not add some credentials to start your new digital me?',
      
Weet niet of we dat ook in het Nederlands willen of wat formeler willen weergeven?

Comment (Norlock) [reply src/i18n/nl/index.ts]: Hmm ja ik vind het in het engels sowieso wat minder irritant overkomen, komt misschien qua taal beter bij past ofzo ;). @JelleMillenaar wat vind jij?

File: src/i18n/nl/index.ts [hunk 11]
```diff
@@ -87,89 +87,89 @@ const nl = {
...
+        CHARACTERS: 'Karakters',
       },
     },
     RESET_APP: {
-      TITLE: '',
-      DESCRIPTION: '',
-      CONFIRM: '',
-      CANCEL: '',
+      TITLE: 'Opnieuw instellen app',
+      DESCRIPTION: 'Weet je zeker dat je de app wilt resetten en alle gegevens wilt verwijderen? ',
```
Comment (Norlock) [inline src/i18n/nl/index.ts]: app opnieuw wilt instellen.

**Review by Norlock — COMMENTED** [review]
laatste puntje daarna is het wel goed denk ik ;)

**Review by Norlock — DISMISSED** [review]

File: src/i18n/nl/index.ts (hunk 1 above)
Comment (nanderstabel) [reply src/i18n/nl/index.ts]: Personally to me `'Het is hier een beetje stil'` is a great translation here from the original English version that came from the UX designs.

In general if the original English version is informal, imo all the translations should reflect the informalness as much as possible.

File: src/i18n/nl/index.ts (hunk 2 above)
Comment (nanderstabel) [reply src/i18n/nl/index.ts]: imo `'Waarom voeg je geen nieuwe credentials toe om te starten met je digitale "me"?'` captures the same kind of informalness/playfulness as `'Why not add some credentials to start your new digital me?'`

File: src/i18n/nl/index.ts [hunk 12]
```diff
@@ -87,89 +87,89 @@ const nl = {
...
+        CHARACTERS: 'Karakters',
       },
     },
     RESET_APP: {
-      TITLE: '',
-      DESCRIPTION: '',
-      CONFIRM: '',
-      CANCEL: '',
+      TITLE: 'App opnieuw instellen',
+      DESCRIPTION: 'Weet je zeker dat je de app opnieuw wilt instellen en alle gegevens wilt verwijderen? ',
```
Comment (nanderstabel) [inline src/i18n/nl/index.ts]: An empty space `' '` snuck in here at the end of the string :) 
`...verwijderen? '` should be `...verwijderen?'`

**Review by nanderstabel — CHANGES_REQUESTED** [review]
Looks good :) 

There seem to be a bunch of translations missing though in the ONBOARDING. Will those be added in a separate PR?

And a general remark from me is that for any translations we will do in the future, we need to stay as close as possible to the original UX designs that you're working with in regards to informalness/playfulness. So I think two of your original translations are closer to the original design and therefore preferred.

File: src/i18n/nl/index.ts (hunk 7 above)
Comment (nanderstabel) [reply src/i18n/nl/index.ts]: I don't think "Opslaan" fits here. 

Original English version is "Select a profile picture", so a better translation imo would be something like:
* "Selecteer"
* "Selecteer een nieuwe afbeelding"

image

File: src/i18n/nl/index.ts [hunk 13]
```diff
@@ -9,167 +9,167 @@ const nl = {
...
     },
     CUSTOMIZE: {
-      NAVBAR_TITLE: '',
+      NAVBAR_TITLE: 'Personalisatie',
       NAME: {
-        TITLE_1: '',
-        TITLE_2: '',
-        SUBTITLE: '',
-        INPUT_PLACEHOLDER: '',
+        TITLE_1: 'Laten we gaan! Kies een',
```
Comment (nanderstabel) [inline src/i18n/nl/index.ts]: What about 'Laten we beginnen! Kies een'?

**Review by nanderstabel — CHANGES_REQUESTED** [review]
Thanks for adding the on-boarding! Just left one minor suggestion :)

**Review by nanderstabel — DISMISSED** [review]

**Review by nanderstabel — APPROVED** [review]
LGTM

**Pull request merged**